### DISCUSSION TITLE
Horizontal mode for tablets

### DISCUSSION THREAD

**Discussion by relliv** [dailydotdev/daily#657] [Feature Requests]
Hi, I am using daily.dev as PWA app with a android tablet but I can't see homepage as horizontal view. Horizontal view works only in article pages. Generally tablet stands are horizontal. So, I want to oriantitation option for mobile devices.

**Comment by idoshamun** [comment]
Hi @EgoistDeveloper,

We need to support it, no doubt. We'll look into it. Thanks for reporting :)
